hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/metadata/SCMDBDefinition.java

```diff
@@ -73,13 +75,30 @@
 
   public static final DBColumnFamilyDefinition<ContainerID, ContainerInfo>
       CONTAINERS =
-      new DBColumnFamilyDefinition<ContainerID, ContainerInfo>(
+      new DBColumnFamilyDefinition<>(
           "containers",
           ContainerID.class,
           new ContainerIDCodec(),
           ContainerInfo.class,
           new ContainerInfoCodec());
 
+  public static final DBColumnFamilyDefinition<Long, CRLInfo> CRL_INFO =
+      new DBColumnFamilyDefinition<>(
+          "crlInfo",
+          Long.class,
+          new LongCodec(),
+          CRLInfo.class,
+          new CRLInfoCodec());
+
+  public static final DBColumnFamilyDefinition<String, Long>
+      CRL_SEQUENCE_ID =
```

Comment:
It's a bit overkill to use a table to just save the crl sequence id. Any chance to put it into the SCM version file?

Reply:
Good suggestion @xiaoyuyao. But, having this sequence id stored in RocksDB gives us the ability to batch transactions when revoke certificates operation is executed. If we use the version file to keep track of this id, we will lose this ability and might cause conflicts with the CRL sequence id.

Reply:
Good point. I'm OK with CRL sequence id in the table for atomic CRL operations using rocksDB.